binderhub/static/js/src/image.js

```diff
@@ -8,6 +9,10 @@ export default function BinderImage(providerSpec) {
 
 BinderImage.prototype.fetch = function() {
   var apiUrl = BASE_URL + "build/" + this.providerSpec;
+  if (BUILD_TOKEN) {
+      apiUrl = apiUrl + `?build_token=${BUILD_TOKEN}`;
```

Comment:
TIL: we can't set headers for `EventSource` requests and that there is a thing called "readable streams" based on `fetch()` https://developer.mozilla.org/en-US/docs/Web/API/Streams_API/Using_readable_streams

Sounds like this is the replacement/"do everything and more that EventStream can do" for event streams (based on GH comments by Chrome devs ;) )